api/src/data/entities/amr.clj

```diff
@@ -13,12 +14,19 @@
                                              (into {} (update instance :pos keyword)))})
                               frames)}))
 
+(defn list-package [package]
+  (let [abs-path (-> (io/file package)
```

Comment:
fix formatting ')' at the end also `(-> package (io/file) (.getParent))` that is we use idiom where all functions are in between parentheses